tox.ini

```diff
@@ -130,7 +130,8 @@ commands=
     pytest astropy/astropy/io/misc/asdf --open-files --run-slow --remote-data \
         -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.types" \
         -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf._types" \
-        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.extension"
+        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.extension" \
+        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.entry_points"
```

Comment:
I think the merger of #1431 will change these, please rebase to pick those changes up.